imap_processing/lo/l0/lo_science.py

```diff
@@ -0,0 +1,147 @@
+"""Processing function for Lo Science Data."""
+
+from collections import namedtuple
+
+import numpy as np
+import xarray as xr
+
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
+from imap_processing.lo.l0.utils.bit_decompression import (
+    DECOMPRESSION_TABLES,
+    Decompress,
+    decompress_int,
+)
+
+HistPacking = namedtuple(
+    "HistPacking",
+    [
+        "bit_length",
+        "section_length",
+        "shape",  # (azimuth, esa_step)
+    ],
+)
+
+hist_data_meta = {
```

Comment:
This global constant name should be all capitalized per PEP8: https://peps.python.org/pep-0008/#constants